workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSBehaviourManager.cs

```diff
@@ -2,32 +2,28 @@
 using Improbable.Worker;
 using UnityEngine;
 
-namespace Improbable.Gdk.Core
+namespace Improbable.Gdk.Core.MonoBehaviours
 {
-    public class ReaderWriterImpl
-    {
-        // This is placeholder code.
-    }
 
     /// <summary>
     ///     Invokes dispatcher callbacks on GameObjects that represent entities.
     /// </summary>
-    public class SpatialOSBehaviourManager
+    internal class SpatialOSBehaviourManager
```

Comment:
Why are these changes mixed up with the namespace move? That should be its own PR.

Reply:
Fair point. The original intention was to just remove the placeholder ReaderWriterImpl from SpatialOSBehaviourManager because that was also a 100% obvious refactor, but since the replacing IReaderInternal is internal, that cascaded into making a few additional things internal. Anyway that will probably end up being done in a PR about the SpatialOSBehaviourManager now that this one is postponed.